app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:chef_id])
```

Comment:
You were on the right track with a more RESTful option being `chefs/ingredients_controller` or `chef_ingredients_controller` -- we want to reserve the "ingredients#index" for listing all the ingredients in the db, not just those for a given chef, so this option is not quite as RESTful. 

Will try to reproduce the issue you had with the folders in the spec directory. As long as they are in the `features` directory, it's okay if they don't exactly match the directory organization of the controller directory